admin/load.php

```diff
@@ -636,3 +639,33 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function to handle admin custom script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_custom_script() {
+	?>
+<script>
+( function( $ ) {
+	if ( window.pagenow === 'settings_page_perflab-modules' ) {
+		// Only listen for ajaxComplete if this is the performance setting page.
+		$( document ).ajaxComplete( function( event, xhr, settings ) {
+			// Check if this is the 'install-plugin' request.
+			if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=install-plugin' ) ) {
+				setTimeout( function() {
+					var plugin_url = $( event.target.activeElement ).attr( 'href' );
```

Comment:
Just to be safe, we should probably add a check here for if there's actually a value. It could be that the target element is _not_ the `a[href]` tag, in which case we need to bail early and do nothing.

Reply:
Thanks for raising. Added check for a href in https://github.com/WordPress/performance/pull/900/commits/35a69fcc2831d5f4ac0e1e69ece748691c8ab5d2